### ISSUE TITLE
RFC5849 support for section 3.5.2

### ISSUE THREAD

**Issue by ga083** [mw99/OhhAuth#1]
Added support for sending the oauth parameters in the form-enconded body instead of just the 'authorization' header.

.diff file and changed OhhAuth.swift file attached in case you want to update the project:
[form-encoded oauth parameters change.zip](https://github.com/mw99/OhhAuth/files/1275961/form-encoded.oauth.parameters.change.zip)

**Comment by mw99** [comment]
Very nice, thanks.

In line 211 you just merge the two parameter dictionaries into one, so the resulting parameters would be in random order. However RFC5849 section 3.5.2 says they must appended. 

> The entity-body MAY include other request-specific parameters, in which case, the protocol parameters SHOULD be appended following the request-specific parameters, properly separated by an "&" character (ASCII code 38).

I have seen OAuth 1 servers to be very strict with parameter ordering before, so that could be a problem. However I have just glanced over your code, so I could be wrong. I will take a closer look when I have the time. Thanks!!

**Comment by ga083** [comment]
Good catch, looks like my server is more permissive than others. I'll take a look.

**Comment by ga083** [comment]
Updated the code to sort the oauth parameters according to 3.5.2.
[sorted form-encoded oauth parameters change.zip](https://github.com/mw99/OhhAuth/files/1283369/sorted.form-encoded.oauth.parameters.change.zip)
